ci/mobile.groovy

```diff
@@ -0,0 +1,102 @@
+common = load 'ci/common.groovy'
+
+def uploadArtifact() {
+  def artifact_dir = pwd() + '/android/app/build/outputs/apk/release/'
+  println (artifact_dir + 'app-release.apk')
+  def artifact = (artifact_dir + 'app-release.apk')
+  def server = Artifactory.server('artifacts')
+  def filename = "im.status.ethereum-${GIT_COMMIT.take(6)}-n-fl.apk"
+  def newArtifact = (artifact_dir + filename)
+  sh "cp ${artifact} ${newArtifact}"
+  def uploadSpec = '{ "files": [ { "pattern": "*apk/release/' + filename + '", "target": "nightlies-local" }]}'
+  def buildInfo = server.upload(uploadSpec)
+  return 'http://artifacts.status.im:8081/artifactory/nightlies-local/' + filename
+}
+
+def prepDeps() {
+  sh 'rm -rf node_modules'
```

Comment:
This can extend the build significantly. I would try to use `npm install` and `npm ls` commands to verify that installed packages have correct versions.

Reply:
That's true, but the purpose of this PR is NOT to fix the build process, it is to fix the Jenkinsfiles and extract steps into re-usable methods. If you want to fix those things, we can do it in a separate PR.